rackcity/views/change_plan_views.py

```diff
@@ -254,3 +267,99 @@ def change_plan_detail(request, id):
         },
         status=HTTPStatus.OK
     )
+
+
+@api_view(['POST'])
+@permission_classes([IsAuthenticated])
+def change_plan_execute(request):
```

Comment:
not sure if i missed this, but we also need to autogenerate Asset  number for AssetCP's without an asset number

Reply:
they are autogenerated on asset.save() for Asset

Reply:
which is called in get_updated_asset() helper method